test/unit/ui/map.test.js

```diff
@@ -628,6 +628,13 @@ test('Map', (t) => {
         t.end();
     });
 
+    t.test('throw on maxZoom smaller than minZoom at init', (t) => {
+        t.throws(() => {
+            createMap({minZoom:10, maxZoom:5});
+        });
```

Comment:
We should test the error message mentions `maxZoom` and `minZoom`. Otherwise, we might pass because of an error line `createMap is undefined`.